src/SqlClient.Tests/SqlClient.Tests.fsproj

```diff
@@ -40,9 +40,7 @@
         <Content Include="connectionStrings.config" />
     </ItemGroup>
     <ItemGroup>
-        <Reference Include="FSharp.Data.SqlClient">
-            <HintPath>../../src/SqlClient/bin/Debug/netstandard20/FSharp.Data.SqlClient.dll</HintPath>
-        </Reference>
+        <ProjectReference Include="../SqlClient/SqlClient.fsproj" />
```

Comment:
we can take a project reference instead of a hintpath so that we always reference the correct targetframework version of the library